charts/app-config/values-integration.yaml

```diff
@@ -143,16 +143,20 @@ govukApplications:
       hosts:
         - name: assets-origin.{{ .Values.k8sExternalDomainSuffix }}
           paths:
+            - path: /assets/
```

Comment:
I think this is ok even though Whitehall has an Ingress in the same ALB group (`assets-origin`) with a path rule for `/assets/whitehall`, because the `whitehall-frontend` Ingress has a higher priority (`group.order: 10` vs our `group.order: 20` here).

I'd recommend checking thoroughly that it's doing what you want after you've rolled this out! :)